projects/common/src/time/time-range.service.test.ts

```diff
@@ -88,4 +116,30 @@ describe('Time range service', () => {
       ['time']: new FixedTimeRange(new Date(1642296703000), new Date(1642396703000)).toUrlString()
     });
   });
+
+  test('Sets time range with navigation events', () => {
```

Comment:
@anandtiwary or someone else, would you be able to take a look at this test ? I'm not 100% confident with it, note the comment